src/commands/tsgen.ts

```diff
@@ -88,7 +89,7 @@ export default class TypeScriptCodeGeneratorCommand extends Command {
         this.log('No Content Types exist in the Stack.')
       }
     } catch (error) {
-      this.error(error, {exit: 1})
+      this.error(error as string, {exit: 1})
```

Comment:
@aman19K Are we sure the error type will be a string.? I suspect it would be Object. If you are not sure use unknown/any type.

Reply:
Yeah, error should be any type. Fixed